backend/src/routes/person.js

```diff
@@ -34,6 +34,128 @@ router.get("/:id/news", async(req, res) => {
     }
 });
 
+router.get("/:id/news/:newsId", async (req, res) => {
```

Comment:
What is the purpose of this route? We already have a route to get news connected to a person whom is a player or staff. 
If we only want one, we could create a news route and just call `/news/id`

Reply:
According to Jerome you cant use :id twice in same route. To wiew specific news

Reply:
the route should be club/:id/news/:newsId